resources/sts/4.14/hypershift/sts_hcp_worker_instance_permission_policy.json

```diff
@@ -8,6 +8,26 @@
                 "ec2:DescribeRegions"
             ],
             "Resource": "*"
+        },
```

Comment:
Adding these to the 4.14, 4.15 and 4.16 files aren't required, just needs to be added to 4.17. The HCP policies aren't specific to each version, so we just need it in the latest. It's a bit confusing, but it's just a quirk of how we manage HCP policy definitions